src/apps/competitions/unpackers/v1.py

```diff
@@ -93,33 +94,46 @@ def _unpack_phases(self):
                     new_phase['end'] = get_datetime(end)
                 else:
                     new_phase['end'] = None
-            if not phase.get('is_parallel_parent') and not phase.get('parent_phasenumber'):
-                task_index = len(self.competition['tasks'])
-                new_phase['tasks'] = [task_index]
-                self.competition['phases'].append(new_phase)
-
-                new_task = {
-                    'name': f'{new_phase["name"]} Task',
-                    'description': new_phase['description'],
-                    'created_by': self.creator.id,
-                    'ingestion_only_during_scoring': phase.get('ingestion_program_only_during_scoring', False)
-                }
-
-                for file_type in ['ingestion_program', 'input_data', 'scoring_program', 'reference_data']:
-                    if file_type in phase:
-                        new_task[file_type] = {
-                            'file_name': phase[file_type],
-                            'file_path': os.path.join(self.temp_directory, phase[file_type]),
-                            'file_type': file_type,
-                            'creator': self.creator.id,
-                        }
-                self.competition['tasks'][task_index] = new_task
-            else:
-                raise CompetitionUnpackingException('Parallel Parents not supported')
+
+            task_index = len(self.competition['tasks'])
+            new_phase['tasks'] = [task_index]
+            self.competition['phases'].append(new_phase)
+
+            new_task = {
+                'name': f'{new_phase["name"]} Task',
+                'description': new_phase['description'],
+                'created_by': self.creator.id,
+                'ingestion_only_during_scoring': phase.get('ingestion_program_only_during_scoring', False)
+            }
+
+            for file_type in ['ingestion_program', 'input_data', 'scoring_program', 'reference_data']:
+                if file_type in phase:
+                    new_task[file_type] = {
+                        'file_name': phase[file_type],
+                        'file_path': os.path.join(self.temp_directory, phase[file_type]),
+                        'file_type': file_type,
+                        'creator': self.creator.id,
+                    }
+            self.competition['tasks'][task_index] = new_task
 
         self._validate_phase_ordering()
         self._set_phase_statuses()
 
+    def _preprocess_phases(self, phases):
```

Comment:
nice! I like this as an interface to modify phases, cool pattern

Reply:
thanks